src/bin/commands/containers.rs

```diff
@@ -149,6 +149,18 @@ pub enum Volumes {
     Remove(RemoveVolume),
 }
 
+macro_rules! volumes_get_field {
+    ($self:ident, $field:ident $(.$cb:ident)?) => {{
```

Comment:
To avoid a lot of redundant code, I added macros so we can extract a single field from an enumeration. The callback allows to add `as_deref` if needed.